casts/cards/src/UICollectionCard/usePreload/usePreload.web.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { Image as RNImage } from 'react-native';
+import type { Content } from '../../types';
+
+export const usePreload = (content: Content[]): void => {
+    return React.useEffect(() => {
+        if (content.length === 0) {
+            /**
+             * Nothing to preload
+             */
+            return;
+        }
+
+        content.forEach((contentItem: Content): void => {
+            if (
+                contentItem.contentType === 'Image' &&
+                contentItem.source &&
+                contentItem.source.uri
+            ) {
+                RNImage.prefetch(contentItem.source.uri);
```

Comment:
AFAIR `react-native-fast-image` have there own cache, are you sure that it's needed?

Reply:
Oh, I see now that it's for a web. Maybe we should implement it in UIImage then?

Reply:
The main purpose is not to add caching, but to preload data for whole list of images. Using this in UIImage won't achieve the goal.

Reply:
I meant not to implement this in UIImage but rather create a method `UIImage.prefetch` that hides platform dependent logic inside (because usage of RNImage on web, is kind of a detail that I don't want to leak outside) :)

Reply:
I actually agree with @savelichalex's statement

Reply:
I actually agree with @savelichalex's statement

Reply:
Agree, done